partcad/src/partcad/part_factory_scad.py

```
@@ -50,22 +52,27 @@
             if scad_path is None:
                 raise Exception("OpenSCAD executable is not found. Please, install OpenSCAD first.")
 
-            stl_path = tempfile.mktemp(".stl")
-            p = await asyncio.create_subprocess_exec(
-                *[
+            with pc_tracer.start_as_current_span(
+                "PartFactoryScad.instantiate.*{asyncio.create_subprocess_exec}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)